### ISSUE TITLE
Relion5 Beta DynaMight deformed backprojection error

### ISSUE THREAD

**Issue by JJJanelia** [3dem/relion#1050]
Hi,

We are grateful for the release of Relion5 Beta and the new features.

While running the final deformed backprojection step of DynaMight job, the following error occured. We are wondering what could be causing it:

UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at ../aten/src/ATen/native/TensorShape.cpp:3483.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]

Thanks in advance!

**Comment by biochem-fan** [comment]
> UserWarning

This is a warning, not an error. Did the job crash after printing it out? Didn't you see other messages?

**Comment by JJJanelia** [comment]
Hi, thanks for kindly looking into this.

These were immediately above the warning message above:

/home/NNNN@NNN.NNN/miniconda3/envs/relion-5.0/lib/python3.10/site-packages/dynamight/deformable_backprojection/backprojection_utils.py:128: UserWarning: Using padding='same' with even kernel lengths and odd dilation may require a zero-padded copy of the input be created (Triggered internally at ../aten/src/ATen/native/Convolution.cpp:1003.)
  sm_bin_mask = torch.nn.functional.conv3d(bin_mask.unsqueeze(0).unsqueeze(0),
/home/NNNN@NNN.NNN/miniconda3/envs/relion-5.0/lib/python3.10/site-packages/torch/functional.py:504:

**Comment by JJJanelia** [comment]
Any help on this would be very much appreciated!
